backend/src/backend/core.clj

```diff
@@ -15,11 +15,13 @@
 
 (ns backend.core
   (:require [ring.adapter.jetty :refer [run-jetty]]
-            [clojure.tools.cli :refer [parse-opts]]
+            [clojure.tools.cli :as cli]
             [aero.core :refer (read-config)]
             [backend.routes.netjson :refer [app-routes]])
   (:gen-class))
 
+(def cfg (read-config "config.edn"))
```

Comment:
This could throw an exception if config.edn doesn't exist.

Reply:
This is intended

Reply:
Does that make sense? If I want to use another configuration file, won't it throw an exception?